modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenu.testcase

```diff
@@ -40,215 +66,246 @@ definition {
-			// Then configuration is persisted to first time signed in user
-			// // assert body element does not have "c-prefers-link-underline" class present
-			// //open accessibility menu
-			// // assert configuration is OFF state
-			// Note: may need to remove any underline html attributes. see https://www.w3schools.com/howto/howto_js_remove_class.asp
 
-		}
+		//task ("Given When Then") {
+
+		// // Given create a new user
+		// // And as a guest user
+		// // // sign out of admin role to become a guest user role
+		// // And toggle show underline to OFF state
+		// // //click show underline to OFF state
+		// // When sign in as a new user for first time
+		// // Then configuration is persisted to first time signed in user
+		// // // assert body element does not have "c-prefers-link-underline" class present
+		// // //open accessibility menu
+		// // // assert configuration is OFF state
+		// // Note: may need to remove any underline html attributes. see https://www.w3schools.com/howto/howto_js_remove_class.asp
+
+		//}
+
 	}
 
 	@description = "LPS-178192. Verifies ON state can persist to first login user."
 	@ignore = "Test Stub"
 	@priority = 3
 	test GuestConfigurationOnStateWillPersistForFirstLogin {
-		task ("Given When Then") {
-
-			// Given create a new user
-			// And as a guest user
-			// // sign out of admin role to become a guest user role
-			// And toggle show underline to ON state
-			// //click show underline to ON state
-			// When sign in as a new user for first time
-			// Then configuration is persisted to first time signed in user
-			// // assert body element has "c-prefers-link-underline" class present
-			// //open accessibility menu
-			// // assert configuration is ON state
-			// Note: may need to remove any underline html attributes. see https://www.w3schools.com/howto/howto_js_remove_class.asp
 
-		}
+		//task ("Given When Then") {
+
+		// // Given create a new user
+		// // And as a guest user
+		// // // sign out of admin role to become a guest user role
+		// // And toggle show underline to ON state
+		// // //click show underline to ON state
+		// // When sign in as a new user for first time
+		// // Then configuration is persisted to first time signed in user
+		// // // assert body element has "c-prefers-link-underline" class present
+		// // //open accessibility menu
+		// // // assert configuration is ON state
+		// // Note: may need to remove any underline html attributes. see https://www.w3schools.com/howto/howto_js_remove_class.asp
+
+		//}
+
 	}
 
 	@description = "LPS-178192. Verifies accessibility menu quick link is keyboard focusable."
-	@ignore = "Test Stub"
-	@priority = 3
+	@priority = 5
 	test LinkIsKeyboardFocusable {
 		task ("When press tab key from 'Skip to Content' link") {
-
-			// Example:
-
-			//KeyPress(
-			//	key_text = "New Experience",
-			//	locator1 = "Button#ANY",
-			//	value1 = "\TAB");
-
+			AccessibilityMenu.goAccessibilityMenu();
```

Comment:
We want to specifically start from `Skip to Content` link, not from the top of the page/body.